### PR TITLE
DEV-512 - time error

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/views/index.js [hunk 1]
```diff
@@ -518,6 +523,7 @@ define(['app', 'lodash', 'moment','text!./ouical-dialog.html', 'directives/mobi-
         function loadDates() {
 
             if(!_ctrl.confObj){
+              console.log(_ctrl.conference,_ctrl.conferences)
```
Comment (sbilodeau) [inline app/views/index.js]:
```suggestion
```

File: app/directives/reg-open.js [hunk 2]
```diff
@@ -71,14 +71,13 @@ function setScopeProps (props){
...
 function findOpenRegsQuery(){
-
   return {
             q:  {
                   '$or'                      : [ { institution: 'CBD' }, { institution: 'cbd' }],
                   schedule                   : { $exists: true },
-                  StartDate                  : { $gt: { $date: moment() } },
-                  'schedule.sideEvents.start': { $lt: { $date: moment() } },
-                  'schedule.sideEvents.end'  : { $gt: { $date: moment() } },
+                  StartDate                  : { $gt: { $date: moment.utc() } },
```
Comment (sbilodeau) [inline app/directives/reg-open.js:78]: You can use `new Date()`. Date will be serialized to utc anyway

**Review by sbilodeau — CHANGES_REQUESTED** [review]
There is console.log

File: app/directives/reg-open.js (hunk 2 above)
Comment (randyhoulahan) [reply app/directives/reg-open.js:78]: thanks sir

Comment (randyhoulahan) [reply app/directives/reg-open.js:78]: You are correct.  Thanks.

**Pull request closed without merging**